improver/nbhood/nbhood.py

```diff
@@ -323,10 +310,12 @@ def __init__(
             neighbourhood_method, radii, lead_times=lead_times
         )
 
-        methods = {"circular": CircularNeighbourhood, "square": SquareNeighbourhood}
```

Comment:
These base plugins need further tidy up, but I'm going to leave that for a separate PR to avoid complicating this one.